libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -628,6 +644,15 @@ AP_GPS_UBLOX::_parse_gps(void)
                     break;
                 }
             }
+        } else if(_msg_id == MSG_ACK_NACK) {
+            switch(_buffer.ack.clsID) {
+            case CLASS_CFG:
+                switch(_buffer.ack.msgID) {
+                case MSG_CFG_SBAS:
+                    _unconfigured_messages &= ~CONFIG_SBAS;
```

Comment:
on the m8p ive tested on, the SBAS message does no exist. at all. if you try request it you get a NACK.

the only way the sbas packet is currently ignored is via a NACK (this code), or receiving a sbas packet, which will never happen.

so you are proposing a param to disable sbas?